scripts/optimize_bayesopt.py

```diff
@@ -0,0 +1,154 @@
+import subprocess
```

Comment:
Same as "swarm", please some doc on this file: docker-wrapper. To be used for Bayes opt. Run as ....